compiler/modules/nback/number.bal

```diff
@@ -305,9 +305,9 @@ function buildDecimalNegate(llvm:Builder builder, Scaffold scaffold, bir:Decimal
 }
 
 final readonly & map<llvm:IntrinsicFunctionName> binaryIntIntrinsics = {
-    "+": "sadd.with.overflow.i64",
-    "-": "ssub.with.overflow.i64",
-    "*": "smul.with.overflow.i64"
+    "+": "sadd.with.overflow.i64.i64",
```

Comment:
Is this available in LLVM 13?

Reply:
Yes. Created #1150 with just this change